homeworks/olena.ambitska_elena-ambitska/5-logic/homework.js

```diff
@@ -0,0 +1,45 @@
+/**
+ *
+ * @param firstValue string with first number
+ * @param secondValue string with second number
+ * @param operation string with operation symbol
+ * @returns {string|number} result of calculation or error message:
+ * - if firstValue or secondValue is not a number, return 'Enter a number'
+ * - if operation is not '+', '-', '*', '/', return 'Choose a valid operation'
+ * - if the result is more than 100, return 'Result is too big'
+ * - else return number of result: 1 '+' 2 returns 3
+ *
+ */
+
+function calculate(firstValue, secondValue, operation) {
+    let result;
+
+    if (Number.isNaN(firstValue) || Number.isNaN(secondValue)) {
+        return 'Enter a number';
```

Comment:
Should be an error for this case 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/d248bf4f-2a2f-44bf-8e5e-742b1f6c19d8)

Reply:
@vladyslav-yermolin-moc done
<details>

<img width="917" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/74259586/fd7f01ec-f75c-4411-b5a8-a7c9ac3d1282">

 </details>